pom.xml

```diff
@@ -601,6 +601,9 @@
           <groupId>com.cognifide.aet</groupId>
           <artifactId>aet-maven-plugin</artifactId>
           <version>${project.version}</version>
+          <configuration>

```

Comment:
yes, it is used when we want to specify different pattern suite from command line.
then we overwrite this empty value with:
```
-DpatternSuite=demo
```